src/App.js

```diff
@@ -1,26 +1,20 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
 
-export class App extends Component {
-  state = {
-    movies: data,
-  };
+export const App = () => {
+  const [movies, addMovie] = useState(data);
```

Comment:
```suggestion
  const [movies, setMovies] = useState(data);
```
this function can be used not for only adding movies, but also for changing/deleting movies